### PR TITLE
vinegar and a vinegar included food

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Adds vinegar.
Made from 10% acetic acid, 90% water.
Adds picoss-kabob from tg. A food fresh of the streets of mogues, with some delicious pickled carp, an onion, and a chilli.
Vinegar condiment bottle. Sprite could be better but I will be dead before I acknowledge red vinegar as a topping.

## Why It's Good For The Game

Not being able to top french fries with vinegar is a sin.
A nice food belonging to a species is also good.

## Images of changes

image

## Testing

Mixed vinegar.
Bottled vinegar.
Made food.
Ate food.
Now I want kabobs.

## Changelog
:cl:
add: Adds vinegar, made from 10% acetic acid, 90% water.
add: Adds picoss-kabob from tg. A food fresh of the streets of mogues, with some delicious pickled carp, an onion, and a chilli.
add: Adds a vinegar condiment bottle. 
/:cl:

### CODE REVIEW

**Comment by Ionward** [comment]
its almost definitely out of scope but it would be neat to be able to ferment wine in a barrel into it too, even though that would be a lil weird without the distinction between red/white wine (i don’t think barrels work out the box with raw reagents so shrug)

**Review by JimKil3 — APPROVED** [review]

File: code/modules/food_and_drinks/recipes/recipes_grill.dm [hunk 1]
```diff
@@ -148,6 +148,17 @@
...
 	result = /obj/item/reagent_containers/food/snacks/tofukabob
 
+/datum/recipe/grill/picoss_kabob
+	reagents = list("vinegar" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/carpmeat,
+		/obj/item/reagent_containers/food/snacks/carpmeat,
+		/obj/item/reagent_containers/food/snacks/grown/onion,
+		/obj/item/reagent_containers/food/snacks/grown/chili,
+		/obj/item/stack/rods,
```
Comment (DGamerL) [inline code/modules/food_and_drinks/recipes/recipes_grill.dm:158]:
```suggestion
		/obj/item/stack/rods
```

**Review by DGamerL — COMMENTED** [review]
I'm adding Prongles: salt and vinegar next.

File: code/modules/food_and_drinks/food/foods/ethnic.dm [hunk 2]
```diff
@@ -150,3 +150,12 @@
 	trash = /obj/item/stack/rods
 	filling_color = "#FFFEE0"
 	list_reagents = list("nutriment" = 8)
+
+/obj/item/reagent_containers/food/snacks/picoss_kabob
+	name = "picoss-kabob"
+	icon_state = "picoss_skewer"
+	desc = "A popular Moghes streetfood consisting of vinegar-marinated space carp on a skewer with onion and chillis."
```
Comment (Henri215) [inline code/modules/food_and_drinks/food/foods/ethnic.dm:157]:
```suggestion
	name = "picoss-kabob"
	desc = "A popular Moghes streetfood consisting of vinegar-marinated space carp on a skewer with onion and chillis."
	icon_state = "picoss_skewer"
```

**Review by DGamerL — APPROVED** [review]
Prongles 2023

File: code/modules/food_and_drinks/food/condiment.dm [hunk 3]
```diff
@@ -218,7 +219,7 @@
...
+	possible_states = list("ketchup" = list("condi_ketchup", "Ketchup", "You feel more American already."), "capsaicin" = list("condi_hotsauce", "Hotsauce", "You can almost TASTE the stomach ulcers now!"), "soysauce" = list("condi_soysauce", "Soy Sauce", "A salty soy-based flavoring"), "frostoil" = list("condi_frostoil", "Coldsauce", "Leaves the tongue numb in it's passage"), "sodiumchloride" = list("condi_salt", "Salt Shaker", "Salt. From space oceans, presumably"), "blackpepper" = list("condi_pepper", "Pepper Mill", "Often used to flavor food or make people sneeze"), "cornoil" = list("condi_cornoil", "Corn Oil", "A delicious oil used in cooking. Made from corn"), "sugar" = list("condi_sugar", "Sugar", "Tasty spacey sugar!"), "vinegar" = list("condi_mixed", "vinegar", "Perfect for chips, if you're feeling Space British."))
```
Comment (Contrabang) [inline code/modules/food_and_drinks/food/condiment.dm:222]: Ditto here, this line is MASSIVE.
```suggestion
	possible_states = list("ketchup" = list("condi_ketchup", "Ketchup", "You feel more American already."), "capsaicin" = list("condi_hotsauce", "Hotsauce", "You can almost TASTE the stomach ulcers now!"), "soysauce" = list("condi_soysauce", "Soy Sauce", "A salty soy-based flavoring"), "frostoil" = list("condi_frostoil", "Coldsauce", "Leaves the tongue numb in it's passage"), "sodiumchloride" = list("condi_salt", "Salt Shaker", "Salt. From space oceans, presumably"), "blackpepper" = list("condi_pepper", "Pepper Mill", "Often used to flavor food or make people sneeze"), "cornoil" = list("condi_cornoil", "Corn Oil", "A delicious oil used in cooking. Made from corn"), "sugar" = list("condi_sugar", "Sugar", "Tasty spacey sugar!"), "vinegar" = list("condi_mixed", "vinegar", "Perfect for chips, if you're feeling Space British."))
```

**Review by Contrabang — APPROVED** [review]
Code is fine, but someday these lists need to be refactored

**Review by Contrabang — APPROVED** [review]
Alright, the lists cant be static. Looks fully good to me

**Review by S34NW — APPROVED** [review]

**Pull request merged**